### ISSUE TITLE
Crash on start when connected to Adafruit RP2040, RP Pico, Microbit

### ISSUE THREAD

**Issue by Ardo9602** [mu-editor/crash-reports#117]
Adafruit RP2040 with Circuitpython 6.2.0 beta4 and release
Microbit with Micropython

Editor starts fine with no board connected, crashes with any of these connected, connecting before or after program start makes no difference.

Here's the log:
```
2021-05-01 22:58:51,442 - root:265(run) INFO: 

-----------------

Starting Mu 1.1.0.beta.4
2021-05-01 22:58:51,442 - root:266(run) INFO: uname_result(system='Windows', node='DESKTOP-14PAKMJ', release='10', version='10.0.19041', machine='AMD64', processor='Intel64 Family 6 Model 158 Stepping 10, GenuineIntel')
2021-05-01 22:58:51,442 - root:267(run) INFO: Python path: ['C:\\Users\\Admiq\\AppData\\Local\\Programs\\Mu Editor', 'C:\\Users\\Admiq\\AppData\\Local\\Programs\\Mu Editor\\Python\\python38.zip', 'C:\\Users\\Admiq\\AppData\\Local\\Programs\\Mu Editor\\Python\\DLLs', 'C:\\Users\\Admiq\\AppData\\Local\\Programs\\Mu Editor\\Python\\lib', 'C:\\Users\\Admiq\\AppData\\Local\\Programs\\Mu Editor\\Python', 'C:\\Users\\Admiq\\AppData\\Roaming\\Python\\Python38\\site-packages', 'C:\\Users\\Admiq\\AppData\\Local\\Programs\\Mu Editor\\Python\\lib\\site-packages', 'C:\\Users\\Admiq\\AppData\\Local\\Programs\\Mu Editor\\Python\\lib\\site-packages\\setuptools-49.6.0-py3.8.egg', 'C:\\Users\\Admiq\\AppData\\Local\\Programs\\Mu Editor\\Python\\lib\\site-packages\\pip-20.2.2-py3.8.egg', 'C:\\Users\\Admiq\\AppData\\Local\\Programs\\Mu Editor\\Python\\lib\\site-packages\\win32', 'C:\\Users\\Admiq\\AppData\\Local\\Programs\\Mu Editor\\Python\\lib\\site-packages\\win32\\lib', 'C:\\Users\\Admiq\\AppData\\Local\\Programs\\Mu Editor\\Python\\lib\\site-packages\\Pythonwin', 'C:\\Users\\Admiq\\AppData\\Local\\Programs\\Mu Editor\\Python\\lib\\site-packages\\IPython\\extensions']
2021-05-01 22:58:51,457 - root:268(run) INFO: Language code: en_US
2021-05-01 22:58:51,457 - mu.settings:220(load) WARNING: No settings file found at C:\Users\Admiq\AppData\Local\python\mu\settings.json; skipping
2021-05-01 22:58:51,511 - mu.virtual_environment:480(ensure_and_create) INFO: Added log handler.
2021-05-01 22:58:51,645 - mu.virtual_environment:491(ensure_and_create) DEBUG: Checking virtual environment; attempt mu-editor/mu#1.
2021-05-01 22:58:51,645 - mu.virtual_environment:542(ensure_path) INFO: Virtual Environment found at: C:\Users\Admiq\AppData\Local\python\mu\mu_venv-38-20210430-215010
2021-05-01 22:58:51,654 - mu.virtual_environment:555(ensure_interpreter) INFO: Interpreter found at: C:\Users\Admiq\AppData\Local\python\mu\mu_venv-38-20210430-215010\scripts\python.exe
2021-05-01 22:58:51,729 - mu.virtual_environment:379(run_subprocess) DEBUG: Process returned 0; output: 38
2021-05-01 22:58:51,729 - mu.virtual_environment:585(ensure_interpreter_version) INFO: Both interpreters at version 38
2021-05-01 22:58:51,730 - mu.virtual_environment:611(ensure_pip) INFO: Pip found at: C:\Users\Admiq\AppData\Local\python\mu\mu_venv-38-20210430-215010\scripts\pip.exe
2021-05-01 22:58:51,730 - mu.virtual_environment:597(ensure_key_modules) DEBUG: Verifying import of: pgzero
2021-05-01 22:58:51,805 - mu.virtual_environment:379(run_subprocess) DEBUG: Process returned 0; output: 
2021-05-01 22:58:51,805 - mu.virtual_environment:597(ensure_key_modules) DEBUG: Verifying import of: flask
2021-05-01 22:58:52,148 - mu.virtual_environment:379(run_subprocess) DEBUG: Process returned 0; output: 
2021-05-01 22:58:52,148 - mu.virtual_environment:597(ensure_key_modules) DEBUG: Verifying import of: qtconsole
2021-05-01 22:58:52,244 - mu.virtual_environment:379(run_subprocess) DEBUG: Process returned 0; output: 
2021-05-01 22:58:52,245 - mu.virtual_environment:597(ensure_key_modules) DEBUG: Verifying import of: esptool
2021-05-01 22:58:52,393 - mu.virtual_environment:379(run_subprocess) DEBUG: Process returned 0; output: 
2021-05-01 22:58:52,393 - mu.virtual_environment:597(ensure_key_modules) DEBUG: Verifying import of: adafruit_board_toolkit
2021-05-01 22:58:52,487 - mu.virtual_environment:379(run_subprocess) DEBUG: Process returned 0; output: 
2021-05-01 22:58:52,487 - mu.virtual_environment:496(ensure_and_create) INFO: Valid virtual environment found at C:\Users\Admiq\AppData\Local\python\mu\mu_venv-38-20210430-215010
2021-05-01 22:58:52,487 - mu.settings:169(save) DEBUG: Saving to C:\Users\Admiq\AppData\Local\python\mu\venv.json
2021-05-01 22:58:52,489 - mu.virtual_environment:509(ensure_and_create) DEBUG: Emitter: <bound PYQT_SIGNAL display_text of StartupWorker object at 0x000001C8C87FA5E0>; Splash Handler; <SplashLogHandler (INFO)>
2021-05-01 22:58:52,585 - mu.logic:758(__init__) INFO: Setting up editor.
2021-05-01 22:58:52,585 - mu.logic:778(__init__) INFO: Log directory: C:\Users\Admiq\AppData\Local\python\mu\Logs
2021-05-01 22:58:52,585 - mu.logic:779(__init__) INFO: Data directory: C:\Users\Admiq\AppData\Local\python\mu
2021-05-01 22:58:52,585 - mu.logic:793(setup) INFO: Available modes: python, circuitpython, microbit, esp, web, pyboard, debugger, pygamezero, lego, pico
2021-05-01 22:58:52,586 - mu.modes.base:61(get_default_workspace) INFO: Using workspace C:\Users\Admiq\mu_code from settings file
2021-05-01 22:58:52,636 - mu.modes.base:61(get_default_workspace) INFO: Using workspace C:\Users\Admiq\mu_code from settings file
2021-05-01 22:58:52,636 - mu.logic:1521(change_mode) INFO: Workspace directory: C:\Users\Admiq\mu_code
2021-05-01 22:58:52,636 - mu.logic:867(restore_session) DEBUG: <SessionSettings from C:\Users\Admiq\AppData\Local\python\mu\session.json>
2021-05-01 22:58:52,643 - mu.logic:889(restore_session) INFO: Loaded files.
2021-05-01 22:58:52,643 - mu.logic:892(restore_session) INFO: User defined environment variables: []
2021-05-01 22:58:52,643 - mu.logic:897(restore_session) INFO: Minify scripts on micro:bit? False
2021-05-01 22:58:52,675 - mu.modes.base:61(get_default_workspace) INFO: Using workspace C:\Users\Admiq\mu_code from settings file
2021-05-01 22:58:52,675 - mu.logic:1521(change_mode) INFO: Workspace directory: C:\Users\Admiq\mu_code
2021-05-01 22:58:52,872 - mu.logic:943(restore_session) INFO: Starting with blank file.
2021-05-01 22:58:53,593 - root:173(excepthook) ERROR: Unrecoverable error
Traceback (most recent call last):
  File "C:\Users\Admiq\AppData\Local\Programs\Mu Editor\Python\lib\site-packages\mu\logic.py", line 711, in check_usb
    detected = mode.find_devices(with_logging=False)
  File "C:\Users\Admiq\AppData\Local\Programs\Mu Editor\Python\lib\site-packages\mu\modes\base.py", line 423, in find_devices
    device = self.compatible_board(port)
  File "C:\Users\Admiq\AppData\Local\Programs\Mu Editor\Python\lib\site-packages\mu\modes\circuitpython.py", line 286, in compatible_board
    for comport in circuitpython_serial.repl_comports():
  File "C:\Users\Admiq\AppData\Local\Programs\Mu Editor\Python\lib\site-packages\adafruit_board_toolkit\circuitpython_serial.py", line 61, in repl_comports
    port for port in comports() if port.interface.startswith("CircuitPython CDC ")
  File "C:\Users\Admiq\AppData\Local\Programs\Mu Editor\Python\lib\site-packages\adafruit_board_toolkit\circuitpython_serial.py", line 46, in comports
    ports = _list_ports_windows.comports()
  File "C:\Users\Admiq\AppData\Local\Programs\Mu Editor\Python\lib\site-packages\adafruit_board_toolkit\_list_ports_windows.py", line 458, in comports
    return list(iterate_comports())
  File "C:\Users\Admiq\AppData\Local\Programs\Mu Editor\Python\lib\site-packages\adafruit_board_toolkit\_list_ports_windows.py", line 342, in iterate_comports
    info = list_ports_common.ListPortInfo(port_name_buffer.value, skip_link_detection=True)
TypeError: __init__() got an unexpected keyword argument 'skip_link_detection'
2021-05-01 22:58:53,744 - mu.settings:169(save) DEBUG: Saving to C:\Users\Admiq\AppData\Local\python\mu\session.json
2021-05-01 22:58:53,745 - mu.settings:169(save) DEBUG: Saving to C:\Users\Admiq\AppData\Local\python\mu\venv.json
2021-05-01 23:04:32,783 - root:265(run) INFO: 

-----------------

Starting Mu 1.1.0.beta.4
2021-05-01 23:04:32,784 - root:266(run) INFO: uname_result(system='Windows', node='DESKTOP-14PAKMJ', release='10', version='10.0.19041', machine='AMD64', processor='Intel64 Family 6 Model 158 Stepping 10, GenuineIntel')
2021-05-01 23:04:32,784 - root:267(run) INFO: Python path: ['C:\\Users\\Admiq\\AppData\\Local\\Programs\\Mu Editor', 'C:\\Users\\Admiq\\AppData\\Local\\Programs\\Mu Editor\\Python\\python38.zip', 'C:\\Users\\Admiq\\AppData\\Local\\Programs\\Mu Editor\\Python\\DLLs', 'C:\\Users\\Admiq\\AppData\\Local\\Programs\\Mu Editor\\Python\\lib', 'C:\\Users\\Admiq\\AppData\\Local\\Programs\\Mu Editor\\Python', 'C:\\Users\\Admiq\\AppData\\Roaming\\Python\\Python38\\site-packages', 'C:\\Users\\Admiq\\AppData\\Local\\Programs\\Mu Editor\\Python\\lib\\site-packages', 'C:\\Users\\Admiq\\AppData\\Local\\Programs\\Mu Editor\\Python\\lib\\site-packages\\setuptools-49.6.0-py3.8.egg', 'C:\\Users\\Admiq\\AppData\\Local\\Programs\\Mu Editor\\Python\\lib\\site-packages\\pip-20.2.2-py3.8.egg', 'C:\\Users\\Admiq\\AppData\\Local\\Programs\\Mu Editor\\Python\\lib\\site-packages\\win32', 'C:\\Users\\Admiq\\AppData\\Local\\Programs\\Mu Editor\\Python\\lib\\site-packages\\win32\\lib', 'C:\\Users\\Admiq\\AppData\\Local\\Programs\\Mu Editor\\Python\\lib\\site-packages\\Pythonwin', 'C:\\Users\\Admiq\\AppData\\Local\\Programs\\Mu Editor\\Python\\lib\\site-packages\\IPython\\extensions']
2021-05-01 23:04:32,784 - root:268(run) INFO: Language code: en_US
2021-05-01 23:04:32,784 - mu.settings:220(load) WARNING: No settings file found at C:\Users\Admiq\AppData\Local\python\mu\settings.json; skipping
2021-05-01 23:04:32,848 - mu.virtual_environment:480(ensure_and_create) INFO: Added log handler.
2021-05-01 23:04:32,978 - mu.virtual_environment:491(ensure_and_create) DEBUG: Checking virtual environment; attempt mu-editor/mu#1.
2021-05-01 23:04:32,978 - mu.virtual_environment:542(ensure_path) INFO: Virtual Environment found at: C:\Users\Admiq\AppData\Local\python\mu\mu_venv-38-20210430-215010
2021-05-01 23:04:32,987 - mu.virtual_environment:555(ensure_interpreter) INFO: Interpreter found at: C:\Users\Admiq\AppData\Local\python\mu\mu_venv-38-20210430-215010\scripts\python.exe
2021-05-01 23:04:33,080 - mu.virtual_environment:379(run_subprocess) DEBUG: Process returned 0; output: 38
2021-05-01 23:04:33,081 - mu.virtual_environment:585(ensure_interpreter_version) INFO: Both interpreters at version 38
2021-05-01 23:04:33,081 - mu.virtual_environment:611(ensure_pip) INFO: Pip found at: C:\Users\Admiq\AppData\Local\python\mu\mu_venv-38-20210430-215010\scripts\pip.exe
2021-05-01 23:04:33,081 - mu.virtual_environment:597(ensure_key_modules) DEBUG: Verifying import of: pgzero
2021-05-01 23:04:33,180 - mu.virtual_environment:379(run_subprocess) DEBUG: Process returned 0; output: 
2021-05-01 23:04:33,180 - mu.virtual_environment:597(ensure_key_modules) DEBUG: Verifying import of: flask
2021-05-01 23:04:33,629 - mu.virtual_environment:379(run_subprocess) DEBUG: Process returned 0; output: 
2021-05-01 23:04:33,629 - mu.virtual_environment:597(ensure_key_modules) DEBUG: Verifying import of: qtconsole
2021-05-01 23:04:33,745 - mu.virtual_environment:379(run_subprocess) DEBUG: Process returned 0; output: 
2021-05-01 23:04:33,745 - mu.virtual_environment:597(ensure_key_modules) DEBUG: Verifying import of: esptool
2021-05-01 23:04:33,911 - mu.virtual_environment:379(run_subprocess) DEBUG: Process returned 0; output: 
2021-05-01 23:04:33,911 - mu.virtual_environment:597(ensure_key_modules) DEBUG: Verifying import of: adafruit_board_toolkit
2021-05-01 23:04:34,025 - mu.virtual_environment:379(run_subprocess) DEBUG: Process returned 0; output: 
2021-05-01 23:04:34,026 - mu.virtual_environment:496(ensure_and_create) INFO: Valid virtual environment found at C:\Users\Admiq\AppData\Local\python\mu\mu_venv-38-20210430-215010
2021-05-01 23:04:34,026 - mu.settings:169(save) DEBUG: Saving to C:\Users\Admiq\AppData\Local\python\mu\venv.json
2021-05-01 23:04:34,034 - mu.virtual_environment:509(ensure_and_create) DEBUG: Emitter: <bound PYQT_SIGNAL display_text of StartupWorker object at 0x000001BF51BDA5E0>; Splash Handler; <SplashLogHandler (INFO)>
2021-05-01 23:04:34,139 - mu.logic:758(__init__) INFO: Setting up editor.
2021-05-01 23:04:34,139 - mu.logic:778(__init__) INFO: Log directory: C:\Users\Admiq\AppData\Local\python\mu\Logs
2021-05-01 23:04:34,139 - mu.logic:779(__init__) INFO: Data directory: C:\Users\Admiq\AppData\Local\python\mu
2021-05-01 23:04:34,139 - mu.logic:793(setup) INFO: Available modes: python, circuitpython, microbit, esp, web, pyboard, debugger, pygamezero, lego, pico
2021-05-01 23:04:34,139 - mu.modes.base:61(get_default_workspace) INFO: Using workspace C:\Users\Admiq\mu_code from settings file
2021-05-01 23:04:34,206 - mu.modes.base:61(get_default_workspace) INFO: Using workspace C:\Users\Admiq\mu_code from settings file
2021-05-01 23:04:34,206 - mu.logic:1521(change_mode) INFO: Workspace directory: C:\Users\Admiq\mu_code
2021-05-01 23:04:34,206 - mu.logic:867(restore_session) DEBUG: <SessionSettings from C:\Users\Admiq\AppData\Local\python\mu\session.json>
2021-05-01 23:04:34,213 - mu.logic:889(restore_session) INFO: Loaded files.
2021-05-01 23:04:34,213 - mu.logic:892(restore_session) INFO: User defined environment variables: []
2021-05-01 23:04:34,213 - mu.logic:897(restore_session) INFO: Minify scripts on micro:bit? False
2021-05-01 23:04:34,250 - mu.modes.base:61(get_default_workspace) INFO: Using workspace C:\Users\Admiq\mu_code from settings file
2021-05-01 23:04:34,250 - mu.logic:1521(change_mode) INFO: Workspace directory: C:\Users\Admiq\mu_code
2021-05-01 23:04:34,463 - mu.logic:943(restore_session) INFO: Starting with blank file.
2021-05-01 23:04:35,139 - root:173(excepthook) ERROR: Unrecoverable error
Traceback (most recent call last):
  File "C:\Users\Admiq\AppData\Local\Programs\Mu Editor\Python\lib\site-packages\mu\logic.py", line 711, in check_usb
    detected = mode.find_devices(with_logging=False)
  File "C:\Users\Admiq\AppData\Local\Programs\Mu Editor\Python\lib\site-packages\mu\modes\base.py", line 423, in find_devices
    device = self.compatible_board(port)
  File "C:\Users\Admiq\AppData\Local\Programs\Mu Editor\Python\lib\site-packages\mu\modes\circuitpython.py", line 286, in compatible_board
    for comport in circuitpython_serial.repl_comports():
  File "C:\Users\Admiq\AppData\Local\Programs\Mu Editor\Python\lib\site-packages\adafruit_board_toolkit\circuitpython_serial.py", line 61, in repl_comports
    port for port in comports() if port.interface.startswith("CircuitPython CDC ")
  File "C:\Users\Admiq\AppData\Local\Programs\Mu Editor\Python\lib\site-packages\adafruit_board_toolkit\circuitpython_serial.py", line 46, in comports
    ports = _list_ports_windows.comports()
  File "C:\Users\Admiq\AppData\Local\Programs\Mu Editor\Python\lib\site-packages\adafruit_board_toolkit\_list_ports_windows.py", line 458, in comports
    return list(iterate_comports())
  File "C:\Users\Admiq\AppData\Local\Programs\Mu Editor\Python\lib\site-packages\adafruit_board_toolkit\_list_ports_windows.py", line 342, in iterate_comports
    info = list_ports_common.ListPortInfo(port_name_buffer.value, skip_link_detection=True)
TypeError: __init__() got an unexpected keyword argument 'skip_link_detection'
2021-05-01 23:04:35,324 - mu.settings:169(save) DEBUG: Saving to C:\Users\Admiq\AppData\Local\python\mu\session.json
2021-05-01 23:04:35,325 - mu.settings:169(save) DEBUG: Saving to C:\Users\Admiq\AppData\Local\python\mu\venv.json
2021-05-01 23:04:53,316 - root:265(run) INFO: 

-----------------

Starting Mu 1.1.0.beta.4
2021-05-01 23:04:53,316 - root:266(run) INFO: uname_result(system='Windows', node='DESKTOP-14PAKMJ', release='10', version='10.0.19041', machine='AMD64', processor='Intel64 Family 6 Model 158 Stepping 10, GenuineIntel')
2021-05-01 23:04:53,316 - root:267(run) INFO: Python path: ['C:\\Users\\Admiq\\AppData\\Local\\Programs\\Mu Editor', 'C:\\Users\\Admiq\\AppData\\Local\\Programs\\Mu Editor\\Python\\python38.zip', 'C:\\Users\\Admiq\\AppData\\Local\\Programs\\Mu Editor\\Python\\DLLs', 'C:\\Users\\Admiq\\AppData\\Local\\Programs\\Mu Editor\\Python\\lib', 'C:\\Users\\Admiq\\AppData\\Local\\Programs\\Mu Editor\\Python', 'C:\\Users\\Admiq\\AppData\\Roaming\\Python\\Python38\\site-packages', 'C:\\Users\\Admiq\\AppData\\Local\\Programs\\Mu Editor\\Python\\lib\\site-packages', 'C:\\Users\\Admiq\\AppData\\Local\\Programs\\Mu Editor\\Python\\lib\\site-packages\\setuptools-49.6.0-py3.8.egg', 'C:\\Users\\Admiq\\AppData\\Local\\Programs\\Mu Editor\\Python\\lib\\site-packages\\pip-20.2.2-py3.8.egg', 'C:\\Users\\Admiq\\AppData\\Local\\Programs\\Mu Editor\\Python\\lib\\site-packages\\win32', 'C:\\Users\\Admiq\\AppData\\Local\\Programs\\Mu Editor\\Python\\lib\\site-packages\\win32\\lib', 'C:\\Users\\Admiq\\AppData\\Local\\Programs\\Mu Editor\\Python\\lib\\site-packages\\Pythonwin', 'C:\\Users\\Admiq\\AppData\\Local\\Programs\\Mu Editor\\Python\\lib\\site-packages\\IPython\\extensions']
2021-05-01 23:04:53,316 - root:268(run) INFO: Language code: en_US
2021-05-01 23:04:53,316 - mu.settings:220(load) WARNING: No settings file found at C:\Users\Admiq\AppData\Local\python\mu\settings.json; skipping
2021-05-01 23:04:53,377 - mu.virtual_environment:480(ensure_and_create) INFO: Added log handler.
2021-05-01 23:04:53,508 - mu.virtual_environment:491(ensure_and_create) DEBUG: Checking virtual environment; attempt mu-editor/mu#1.
2021-05-01 23:04:53,508 - mu.virtual_environment:542(ensure_path) INFO: Virtual Environment found at: C:\Users\Admiq\AppData\Local\python\mu\mu_venv-38-20210430-215010
2021-05-01 23:04:53,517 - mu.virtual_environment:555(ensure_interpreter) INFO: Interpreter found at: C:\Users\Admiq\AppData\Local\python\mu\mu_venv-38-20210430-215010\scripts\python.exe
2021-05-01 23:04:53,626 - mu.virtual_environment:379(run_subprocess) DEBUG: Process returned 0; output: 38
2021-05-01 23:04:53,626 - mu.virtual_environment:585(ensure_interpreter_version) INFO: Both interpreters at version 38
2021-05-01 23:04:53,627 - mu.virtual_environment:611(ensure_pip) INFO: Pip found at: C:\Users\Admiq\AppData\Local\python\mu\mu_venv-38-20210430-215010\scripts\pip.exe
2021-05-01 23:04:53,627 - mu.virtual_environment:597(ensure_key_modules) DEBUG: Verifying import of: pgzero
2021-05-01 23:04:53,745 - mu.virtual_environment:379(run_subprocess) DEBUG: Process returned 0; output: 
2021-05-01 23:04:53,754 - mu.virtual_environment:597(ensure_key_modules) DEBUG: Verifying import of: flask
2021-05-01 23:04:54,219 - mu.virtual_environment:379(run_subprocess) DEBUG: Process returned 0; output: 
2021-05-01 23:04:54,219 - mu.virtual_environment:597(ensure_key_modules) DEBUG: Verifying import of: qtconsole
2021-05-01 23:04:54,334 - mu.virtual_environment:379(run_subprocess) DEBUG: Process returned 0; output: 
2021-05-01 23:04:54,334 - mu.virtual_environment:597(ensure_key_modules) DEBUG: Verifying import of: esptool
2021-05-01 23:04:54,497 - mu.virtual_environment:379(run_subprocess) DEBUG: Process returned 0; output: 
2021-05-01 23:04:54,497 - mu.virtual_environment:597(ensure_key_modules) DEBUG: Verifying import of: adafruit_board_toolkit
2021-05-01 23:04:54,613 - mu.virtual_environment:379(run_subprocess) DEBUG: Process returned 0; output: 
2021-05-01 23:04:54,613 - mu.virtual_environment:496(ensure_and_create) INFO: Valid virtual environment found at C:\Users\Admiq\AppData\Local\python\mu\mu_venv-38-20210430-215010
2021-05-01 23:04:54,614 - mu.settings:169(save) DEBUG: Saving to C:\Users\Admiq\AppData\Local\python\mu\venv.json
2021-05-01 23:04:54,614 - mu.virtual_environment:509(ensure_and_create) DEBUG: Emitter: <bound PYQT_SIGNAL display_text of StartupWorker object at 0x000001335C92A5E0>; Splash Handler; <SplashLogHandler (INFO)>
2021-05-01 23:04:54,723 - mu.logic:758(__init__) INFO: Setting up editor.
2021-05-01 23:04:54,723 - mu.logic:778(__init__) INFO: Log directory: C:\Users\Admiq\AppData\Local\python\mu\Logs
2021-05-01 23:04:54,724 - mu.logic:779(__init__) INFO: Data directory: C:\Users\Admiq\AppData\Local\python\mu
2021-05-01 23:04:54,724 - mu.logic:793(setup) INFO: Available modes: python, circuitpython, microbit, esp, web, pyboard, debugger, pygamezero, lego, pico
2021-05-01 23:04:54,724 - mu.modes.base:61(get_default_workspace) INFO: Using workspace C:\Users\Admiq\mu_code from settings file
2021-05-01 23:04:54,787 - mu.modes.base:61(get_default_workspace) INFO: Using workspace C:\Users\Admiq\mu_code from settings file
2021-05-01 23:04:54,787 - mu.logic:1521(change_mode) INFO: Workspace directory: C:\Users\Admiq\mu_code
2021-05-01 23:04:54,788 - mu.logic:867(restore_session) DEBUG: <SessionSettings from C:\Users\Admiq\AppData\Local\python\mu\session.json>
2021-05-01 23:04:54,795 - mu.logic:889(restore_session) INFO: Loaded files.
2021-05-01 23:04:54,795 - mu.logic:892(restore_session) INFO: User defined environment variables: []
2021-05-01 23:04:54,795 - mu.logic:897(restore_session) INFO: Minify scripts on micro:bit? False
2021-05-01 23:04:54,832 - mu.modes.base:61(get_default_workspace) INFO: Using workspace C:\Users\Admiq\mu_code from settings file
2021-05-01 23:04:54,832 - mu.logic:1521(change_mode) INFO: Workspace directory: C:\Users\Admiq\mu_code
2021-05-01 23:04:55,030 - mu.logic:943(restore_session) INFO: Starting with blank file.
2021-05-01 23:04:59,769 - mu.logic:1394(show_admin) INFO: Showing admin with logs from C:\Users\Admiq\AppData\Local\python\mu\Logs\mu.log
2021-05-01 23:04:59,769 - mu.virtual_environment:788(installed_packages) INFO: Discovering installed third party modules in venv.
2021-05-01 23:05:00,398 - mu.virtual_environment:193(run) DEBUG: Process output: Package                Version

---------------------- -------

adafruit-board-toolkit 1.0.1

backcall               0.2.0

bitstring              3.1.7

cffi                   1.14.5

click                  7.1.2

colorama               0.4.4

cryptography           3.4.7

decorator              5.0.7

ecdsa                  0.16.1

esptool                3.0

Flask                  1.1.2

ipykernel              5.5.3

ipython                7.22.0

ipython-genutils       0.2.0

itsdangerous           1.1.0

jedi                   0.18.0

Jinja2                 2.11.3

jupyter-client         6.1.12

jupyter-core           4.7.1

MarkupSafe             1.1.1

numpy                  1.20.2

parso                  0.8.2

pgzero                 1.2.1

pickleshare            0.7.5

pip                    21.1.1

prompt-toolkit         3.0.18

pycparser              2.20

pygame                 2.0.1

Pygments               2.8.1

pyserial               3.5

python-dateutil        2.8.1

pywin32                300

pyzmq                  22.0.3

qtconsole              4.7.4

QtPy                   1.9.0

reedsolo               1.5.4

setuptools             56.0.0

six                    1.15.0

tornado                6.1

traitlets              5.0.5

wcwidth                0.2.5

Werkzeug               1.0.1

wheel                  0.36.2
2021-05-01 23:05:00,398 - mu.virtual_environment:797(installed_packages) INFO: []
```

**Comment by dhalbert** [comment]
Please also update to CircuitPython 7.3.0 if you have not already.

**Comment by Ardo9602** [comment]
Hi, sorry for this late reply, I just got to try this today

First I downloaded v1.1.1 from the website and tried it on a  Rpi Pico with
circuitpython 7.3.1, it still crashed
Then I desperately tried the download from github you suggested (also
v1.1.1 ?) and... this works ! no crash!

Thanks so much

**Comment by carlosperate** [comment]
Hi @Ardo9602, thanks for the update!

The installe from the GitHub link in this thread might identify itself as v1.1.1, but it contains some aditional bug fixes for the next release, so when Mu v1.2 is out the installer from the website will work as well.

I'll close this as resolved, as the fix will be present in the next release, thanks again for the report and testing! 👍
